src/views/templates/details/NoEditableTemplateAlert.tsx

```diff
@@ -42,10 +43,10 @@ const NoEditableTemplateAlert: React.FC<NoEditableTemplateAlertProps> = ({ templ
       })}
     >
       <Trans ns="plugin__kubevirt-plugin">
-        {{ osName }} VirtualMachine can not be edited because it is provided by {{ providerName }}
-        Virtualization Operator.
+        {{ osName }} VirtualMachine can not be edited because it is provided by{' '}
+        <Link to={OPENSHIFT_VIRT_PATH}>OpenShift Virtualization Operator.</Link>
         <br />
```

Comment:
```suggestion
        {{ osName }} Template can not be edited because it is provided by{' '}
        OpenShift Virtualization Operator.
```

We do not want to link to the operator page, regular users can't use this link.

Reply:
ok